### ISSUE TITLE
Cleanup and fix failed contact verification heavy tests

### ISSUE THREAD

**Issue by jiangwen84** [idaholab/moose#10165]
## Rationale
We have developed new augmented Lagrangian code and this causes several contact verification heavy tests fail. Those tests need to be updated.  

## Impact
No impact on existing codes.

**Comment by permcody** [comment]
Look who's modifying the template. Please follow the template and keep us NQA-1 ready.

**Comment by andrsd** [comment]
Yeah. This is a problem. We have people that are not NQA-1 trained and they contribute to the code, so they do not know the template is not just a suggestion, but something rather mandatory.

Should we have something like:
```
IMPORTANT! DO NOT EDIT THIS TEMPLATE!
```
on the first line? Would that even help?
